modules/statics/src/coins.ts

```diff
@@ -40,7 +40,19 @@ const EOS_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.SUPPORTS_TOKE
 const SOL_FEATURES = [...AccountCoin.DEFAULT_FEATURES, CoinFeature.REQUIRES_RESERVE, CoinFeature.SUPPORTS_TOKENS];
 const NEAR_FEATURES = [...AccountCoin.DEFAULT_FEATURES];
 
-export const coins = CoinMap.fromCoins([
+// Alorand is migration the name for the tokens, this are the equivalences to support new names during transition
+// STLX-10016 will revert this to support old names when new names were released
+const algorandKeyEquivalences = new Map([
```

Comment:
Can we add a new coin type that's an `aliasCoin` or something like that, which just redirects to another coin name? This approach feels quite hacky.

Reply:
Added field alias to coin definition so it can be also be mapped by its alias. I prefer this change than add a `aliasCoin` to avoid duplication in case of listings.